dist/backdraft.js

```diff
@@ -1275,8 +1275,18 @@ _.extend(Plugin.factory, {
 
     render : function() {
       var cells = this.findCells(), node;
-      _.each(this.columnsConfig, function(config) {
-        node = cells.filter(config.nodeMatcher(config));
+      var cleanConfigs = this.columnsConfig.map(function(currentValue,index){
+        if (index===0) {
```

Comment:
It was meant for the bulk column but now I should check it it's bulk not if it's index 0.